source/shared/net/references/developer-tools.rst

```diff
@@ -0,0 +1,35 @@
+.. _developer-tools:
+
+===============
+Developer Tools
+===============
+
+.. contents::
+   :local:
+   :backlinks: none
+
+
+Decrypt encrypted output
+========================
+
+Some Concordium tools, such as the Mobile Wallet, sometimes produce encrypted output. These files can be decrypted and inspected using the **utils** tool . You can download the tool from **reference here**.
```

Comment:
Is the **reference here** intended?

Reply:
No, just me forgetting to go back and change the placeholder I put in.